### ISSUE TITLE
the bytes `\xf0\x9c\x83\x96` stop the entire output

### ISSUE THREAD

**Issue by SimonHeimberg** [php-censor/php-censor#145]
When the four bytes `\xf0\x9c\x83\x96` are in the output of any step,
all output from there is truncated. Also the output of the following steps is nowhere.

This byte sequence in UTF-8 is for `\U0001c0d6`, a non-existing unicode character (see in [character reference](https://en.wikibooks.org/wiki/Unicode/Character_reference)).

## Expected behavior

The output should be shown normally (probably with some nonprintable characters replaced).
All following results should be there.

## Actual behavior

All output starting from this bytes is truncated. Also the output from following steps.

The other steps are run normally (visible in the `console.log`).

The database contains only what is shown in the log output.

The behaviour is similar to #116.

## Steps to reproduce
insert this into any build config:
```yaml
    Shell:
        - '/usr/bin/printf "aaa \xf0\x9c\x83\x96 bytes before and this are not shown"'
        - echo is not shown
    Shell_:
        - echo is not shown as well
```
only `aaa ` will be shown.
all output after this is lost, even from another build step

or alternative equivalent print command:
- '/usr/bin/printf "aaa \U0001c0d6 bbb"'

## Environment
php-censor: eb1aa43

os: linux (ubuntu 14.04)
php: 7.1.13
mysql: 14.4

**Comment by corpsee** [comment]
@SimonHeimberg Thanks for the detailed explanation! Bug accepted.

**Comment by corpsee** [comment]
Fixed in branch `non-utf-8-fix`. @SimonHeimberg Can you check fix please?

**Comment by corpsee** [comment]
Finally fixed in #146.

**Comment by SimonHeimberg** [comment]
Additional info about UTF_8 in mysql (according to https://dev.mysql.com/doc/refman/5.7/en/charset-unicode-utf8.html):
* No support for supplementary characters (BMP characters only).
* A maximum of **three** bytes per multibyte character.

The problem with the character in this example is, that it is **four** bytes long.
